source/user-manual/capabilities/system-inventory/configuration.rst

```diff
@@ -50,7 +50,8 @@ Where:
 - ``<network>`` enables or disables the network scan. The default value is ``yes``. The allowed values are ``yes`` and ``no``.
 - ``<packages>`` enables or disables the scanning of packages with a default value of ``yes``. The allowed values are ``yes`` and ``no``.
 - ``<ports all="no">`` enables or disables the port scan. The default value is ``yes``. You can configure two allowed values of ``yes`` and ``no``. This option also accepts an additional parameter ``all``, with which you can restrict the scan to only listening ports using ``<ports all="no">``. If you want Syscollector to scan all ports, then you change the value to ``yes``.
-- ``<processes>`` enables or disables the scanning for running processes on a monitored endpoint. The default value of ``yes``. The allowed values are ``yes`` and ``no``.
+- ``<processes>`` enables or disables the scanning for running processes on a monitored endpoint. The default value is ``yes``. The allowed values are ``yes`` and ``no``.
+- ``<hotfixes>`` option hidden by default, enables or disables the scanning of hotfixes for Windows systems only. The default value is ``yes``. The allowed values are ``yes`` and ``no``. This option is ignored for other systems.
```

Comment:
I think the first part is a bit confusing

```suggestion
- ``<hotfixes>`` enables or disables the scanning of hotfixes for Windows systems only. The default value is ``yes``. The allowed values are ``yes`` and ``no``. This option is ignored for other systems.
```

The "for windows systems only" part implies that for other systems this applies

```suggestion
- ``<hotfixes>`` option hidden by default, enables or disables the scanning of hotfixes for Windows systems. The default value is ``yes``. The allowed values are ``yes`` and ``no``. This option is ignored for other systems.
```

Full suggestions
```suggestion
- ``<hotfixes>`` enables or disables the scanning of hotfixes for Windows systems. The default value is ``yes``. The allowed values are ``yes`` and ``no``. This option is ignored for other systems.
```